sys/arvo.hoon

```diff
@@ -502,101 +515,232 @@
     $(ova (weld p.nyx ova), mor (weld q.nyx t.mor))
   --
 --
-=+  pit=`vase`!>(.)                                     ::
-=+  bud=pit                                             ::  becomes tang
-::  =+  vil=(viol p.bud)                                ::  cached reflexives
-=+  vil=(viol p.bud)                                    ::  cached reflexives
-=|  $:  lac/?                                           ::  laconic bit
-        eny/@                                           ::  entropy
+=<  ::  Arvo larval stage
```

Comment:
My gut feeling here is that this larval core is too flexible and accommodating, and that we'll want to lock it down more later. I think we'll get a better sense for that after using this version for a while, though, so I think it's good as it is.

Reply:
That may be, but it's also pretty locked down already. It needs to support installing %zuse for `+ivory`, plus the vanes for `+solid`. Besides that, it only accepts identity and entropy. It has to support `+come` and `+load` for future kernel upgrades, and `+wish` for using `+ivory` on cc-release. I just removed the old %vega reset and `+peek` is now a no-op. I'll keep thinking about further ways to lock it down (it might be nice to use a stricter core variance), but nothing comes to mind at the moment.